preprocessor/src/test/java/com/azure/autorest/PreprocessorTests.java

```diff
@@ -24,10 +25,14 @@ public void processTest() {
         String codeModelFileName = "containerregistry-code-model.yaml";
 
         CodeModel codeModel = preprocessor.loadCodeModel(codeModelFileName);
+        Assert.assertEquals(codeModel.getOperationGroups().size(), 3);
+        Assert.assertEquals(codeModel.getOperationGroups().get(0).getOperations().size(), 15);
+        Assert.assertEquals(codeModel.getOperationGroups().get(1).getOperations().size(), 11);
+        Assert.assertEquals(codeModel.getOperationGroups().get(2).getOperations().size(), 3);
         codeModel = preprocessor.transform(codeModel);
-        String output = preprocessor.dump(codeModel);
+        Assert.assertEquals(codeModel.getOperationGroups().get(0).getOperations().size(), 18); // additional 3 pagination operations
```

Comment:
The count is likely fragile. Maybe just verify several name of the operation exists?

Reply:
Sure!